templates/MetadataGenerator/MD_DataIdentification.xml.twig

```diff
@@ -89,6 +89,13 @@
             </gmd:otherConstraints>
         </gmd:MD_LegalConstraints>
     </gmd:resourceConstraints>
+    <gmd:resourceConstraints xlink:title="Cite As">
```

Comment:
Do we want to move this up? According to the order in the ticket

Reply:
yeah, I will.

Reply:
also merge in develop